services/graphql/src/graphql/definitions/field.js

```diff
@@ -106,6 +134,21 @@ type SelectFieldOption {
   externalIdentifier: String
 }
 
+input CreateBooleanFieldMutationInput {
+  "The internal name of the field."
+  name: String!
+  "The user-facing field label. This is what will appear on a form."
+  label: String!
+  "Whether the field is globally required."
+  required: Boolean = false
+  "Whether the field is currently active."
+  active: Boolean = true
+  "Whether the select field supports multiple answers."
```

Comment:
This comment wouldn't apply to a boolean field, as multiple answers are not supported. How is `value` used by the mutation?

Reply:
Actually, it doesn't look like the `value` field is used by the resolver. Remove

Reply:
@zarathustra323  you are correct and this should now be removed